crates/rustc_codegen_spirv/src/builder/spirv_asm.rs

```diff
@@ -1357,6 +1358,61 @@ impl<'cx, 'tcx> Builder<'cx, 'tcx> {
+            },
+            (OperandKind::FPOperationMode, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::FPOperationMode(x)),
+                Err(()) => self.err(format!("unknown FPOperationMode {word}")),
+            },
+            (OperandKind::OverflowModes, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::OverflowModes(x)),
+                Err(()) => self.err(format!("unknown OverflowModes {word}")),
+            },
+            (OperandKind::PackedVectorFormat, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::PackedVectorFormat(x)),
+                Err(()) => self.err(format!("unknown PackedVectorFormat {word}")),
+            },
+            (OperandKind::HostAccessQualifier, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::HostAccessQualifier(x)),
+                Err(()) => self.err(format!("unknown HostAccessQualifier {word}")),
+            },
+            (OperandKind::CooperativeMatrixOperands, Some(word)) => {
+                match parse_bitflags_operand(COOPERATIVE_MATRIX_OPERANDS, word) {
+                    Some(x) => inst
+                        .operands
+                        .push(dr::Operand::CooperativeMatrixOperands(x)),
+                    None => self.err(format!("Unknown CooperativeMatrixOperands {word}")),
+                }
+            }
+            (OperandKind::CooperativeMatrixLayout, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::CooperativeMatrixLayout(x)),
+                Err(()) => self.err(format!("unknown CooperativeMatrixLayout {word}")),
+            },
+            (OperandKind::CooperativeMatrixUse, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::CooperativeMatrixUse(x)),
+                Err(()) => self.err(format!("unknown CooperativeMatrixUse {word}")),
+            },
+            (OperandKind::InitializationModeQualifier, Some(word)) => match word.parse() {
+                Ok(x) => inst
+                    .operands
+                    .push(dr::Operand::InitializationModeQualifier(x)),
+                Err(()) => self.err(format!("unknown InitializationModeQualifier {word}")),
+            },
+            (OperandKind::LoadCacheControl, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::LoadCacheControl(x)),
+                Err(()) => self.err(format!("unknown LoadCacheControl {word}")),
+            },
+            (OperandKind::StoreCacheControl, Some(word)) => match word.parse() {
+                Ok(x) => inst.operands.push(dr::Operand::StoreCacheControl(x)),
+                Err(()) => self.err(format!("unknown StoreCacheControl {word}")),
+            },
+            (OperandKind::LiteralFloat, Some(word)) => todo!(),
```

Comment:
I could copy-paste all of the above, but I have no idea what to do here, as there is no such thing as an `dr::Operand::LiteralFloat`